client/state/plugins/wporg/actions.js

```diff
@@ -128,6 +129,15 @@ export function fetchPluginsList(
 		} )
 			.then( ( { info, plugins } ) => {
 				dispatch( receivePluginsList( category, page, searchTerm, plugins, null, info ) );
+				// Do not trigger a new tracks event for subsequent pages.
+				if ( searchTerm && info?.page === 1 ) {
```

Comment:
Do we want to track when the user has received results for their search, or when the user has searched for something? I realize the difference is nuanced, but still worth confirming we're tracking the intentional behavior.

Reply:
We want to track **what** users are searching and the **number** of results they get.

Reply:
But won't this event be triggered only if the search has yielded successful results? Or that's intentional and we only care about meaningful search results that yield results?

Reply:
This will run even for searches with 0 results. It won't run though if the API doesn't respond or returns an error which is something we are not interested at the moment but seems like a good candidate for a future iteration.